src/gpio.rs

```diff
@@ -98,6 +98,37 @@ pub struct AF14;
 /// Alternate function 15 (type state)
 pub struct AF15;
 
+macro_rules! doc_comment {
```

Comment:
This is taken from https://github.com/rust-lang/rust/blob/4a365a29d64bec75d107214319a129ba68fc12a3/src/libcore/num/mod.rs#L28-L33